docs/INSTALLATION.md

```diff
@@ -91,31 +108,39 @@ See [Intel Fortran](###-Intel-Fortran)
 
 ### Intel Fortran
 
-For [Intel Fortran](https://software.intel.com/en-us/fortran-compilers) you have to first initialize the fortran compiler. On windows with Intel Fortran 2019 and Visual Studio 2017 the command line would be:
+For [Intel Fortran](https://software.intel.com/en-us/fortran-compilers)
+you have to first initialize the fortran compiler. On windows with Intel
+Fortran 2019 and Visual Studio 2017 the command line would be:
 
 ```Batchfile
 "c:\Program Files (x86)\IntelSWTools\compilers_and_libraries_2019\windows\bin\ifortvars.bat" intel64 vs2017
 ```
 
-This sources the paths for Intel Fortran and cmake should pick it up correctly. Also, on windows you should specify the cmake generator `NMake` for a command line build, eg.
+This sources the paths for Intel Fortran and cmake should pick it up
+correctly. Also, on windows you should specify the cmake generator
+`NMake` for a command line build, eg.
 
 ```Batchfile
 mkdir Debug
 cd Debug
-cmake -G"NMake Makefiles" -DCMAKE_BUILD_TYPE="Debug" ..
+cmake -G"NMake Makefiles" ..
 NMake
 ctest -V
 ```
 
-The commands above will create a Debug directory (not necessary, but good practice) and build (NMake) the executables and test them (ctest).
+The commands above will create a Debug directory (not necessary, but
+good practice) and build (NMake) the executables and test them (ctest).
 
-For other versions of Intel Fortran you want to search your installation for `ifortvars.bat` on windows and on linux/macOS `ifortvars.sh`. Execute the script in a shell without options and a help will explain which options you have. On linux/macOS the commands would be:
+For other versions of Intel Fortran you want to search your installation
+for `ifortvars.bat` on windows and on linux/macOS `ifortvars.sh`.
```

Comment:
Linux, Mac and Windows are formal names, and so should be capitalized.  Probably remove the `slash` as these aren't strictly related things.